packages/server/tests/acceptance/rpc_batch1.spec.ts

```diff
@@ -992,6 +992,180 @@ describe('@api-batch-1 RPC Server Acceptance Tests', function () {
         await Assertions.assertPredefinedRpcError(error, sendRawTransaction, true, relay, [signedTx, requestDetails]);
       });
 
+      it('@xts should fail "eth_sendRawTransaction" for HBAR crypto transfer to zero addresses', async function () {
+        const sendHbarTx = {
+          ...defaultLegacyTransactionData,
+          value: ONE_TINYBAR,
+          to: ethers.ZeroAddress,
+          nonce: await relay.getAccountNonce(accounts[1].address),
+          gasPrice: await relay.gasPrice(),
+        };
+
+        const signedSendHbarTx = await accounts[1].wallet.signTransaction(sendHbarTx);
+        const error = predefined.INTERNAL_ERROR('Transaction execution returns a null value');
+
+        await Assertions.assertPredefinedRpcError(error, sendRawTransaction, true, relay, [
+          signedSendHbarTx,
+          requestDetails,
+        ]);
+      });
+
+      // https://github.com/hiero-ledger/hiero-consensus-node/blob/main/hedera-node/docs/system-accounts-operations.md
+      const hederaReservedAccounts = [
+        // system accounts (≤ 0.0.750) - should return INVALID_CONTRACT_ID
+        {
+          address: '0x0000000000000000000000000000000000000002',
+          description: '0.0.2 treasury',
+          expectedError: 'INVALID_CONTRACT_ID',
+        },
+        {
+          address: '0x0000000000000000000000000000000000000003',
```

Comment:
should we include a section with Ethereum precompiles?, that is, addresses in the range `0x1` to `0xa` (and also `0xb` in the future)